dags/app_store_analytics.py

```diff
@@ -89,7 +89,7 @@
             task_id=f"{derived_table}_query",
             project_id=PROJECT_ID,
             dataset_id=DERIVED_DATASET_ID,
-            sql_file_path=f"sql/{DERIVED_DATASET_ID}/{derived_table}/query.sql",
+            sql_file_path=f"moz-fx-data-shared-prod/sql/{DERIVED_DATASET_ID}/{derived_table}/query.sql",
```

Comment:
```suggestion
            sql_file_path=f"sql/moz-fx-data-shared-prod/{DERIVED_DATASET_ID}/{derived_table}/query.sql",
```

And similar for the other changes here.